main/exercise/multiple_answer_true_false_degree_certainty.class.php

```diff
@@ -0,0 +1,1151 @@
+<?php
+
+    /* For licensing terms, see /license.txt */
+
+    /**
+     * Class MultipleAnswerTrueFalseDegreeCertainty
+     * This class allows to instantiate an object of type MULTIPLE_ANSWER
+     * (MULTIPLE CHOICE, MULTIPLE ANSWER), extending the class question
+     *
+     * @package chamilo.exercise
+     */
+    class MultipleAnswerTrueFalseDegreeCertainty extends Question {

```

Comment:
It' better to call the file as the class MultipleAnswerTrueFalseDegreeCertainty.php